tldr/hybrid_extractor.py

```
@@ -3151,6 +3171,153 @@ def _extract_elixir_function(self, node, source: bytes, is_private: bool) -> Fun
+
+        self._extract_zig_nodes(tree.root_node, source, module_info)
+        return module_info
+
+    def _get_zig_parser(self) -> "Parser":
+        if "zig" not in self._ts_parsers:
+            zig_lang = Language(tree_sitter_zig.language())
+            parser = Parser(zig_lang)
+            self._ts_parsers["zig"] = parser
+        return self._ts_parsers["zig"]
+
+    def _extract_zig_nodes(self, node, source: bytes, module_info: ModuleInfo, in_type: bool = False) -> None:
```

Ruff (0.14.14)

Boolean-typed positional argument in function definition

(FBT001)